File: MORM.Java/src/MORM/Java/Models/Venda/Transfiscal.java
package MORM.Java.Models.Venda;

import java.util.Date;
import MORM.Java.Classes.*;
import MORM.Java.Classes.Mapping.*;

@TabelaAttribute(Nome = "TRANSFISCAL")
public class Transfiscal extends CollectionItem
{
    @CampoAttribute(Campo = "ID_TRANSACAO", Tipo = CampoTipo.Key)
    public String Id_Transacao;
    @CampoAttribute(Campo = "U_VERSION", Tipo = CampoTipo.Nul)
    public String U_Version;
    @CampoAttribute(Campo = "CD_OPERADOR", Tipo = CampoTipo.Req)
    public Integer Cd_Operador;
    @CampoAttribute(Campo = "DT_CADASTRO", Tipo = CampoTipo.Req)
    public Date Dt_Cadastro;
    @CampoAttribute(Campo = "TP_OPERACAO", Tipo = CampoTipo.Req)
    public Integer Tp_Operacao;
    @CampoAttribute(Campo = "TP_MODALIDADE", Tipo = CampoTipo.Req)
    public Integer Tp_Modalidade;
    @CampoAttribute(Campo = "TP_MODELONF", Tipo = CampoTipo.Req)
    public Integer Tp_Modelonf;
    @CampoAttribute(Campo = "CD_SERIE", Tipo = CampoTipo.Req)
    public String Cd_Serie;
    @CampoAttribute(Campo = "NR_NF", Tipo = CampoTipo.Req)
    public Integer Nr_Nf;
    @CampoAttribute(Campo = "TP_PROCESSAMENTO", Tipo = CampoTipo.Req)
    public String Tp_Processamento;
    @CampoAttribute(Campo = "DS_CHAVEACESSO", Tipo = CampoTipo.Nul)
    public String Ds_Chaveacesso;
    @CampoAttribute(Campo = "DT_RECEBIMENTO", Tipo = CampoTipo.Nul)
    public Date Dt_Recebimento;
    @CampoAttribute(Campo = "NR_RECIBO", Tipo = CampoTipo.Nul)
    public String Nr_Recibo;
}

File: MORM.CSharp/MORM.Dominio/Models/Venda/Transfiscal.cs
using MORM.Util.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("TRANSFISCAL")]
    public class Transfiscal : CollectionItem
    {
        [Campo("ID_TRANSACAO", CampoTipo.Key)]
        public string Id_Transacao { get; set; }
        [Campo("U_VERSION", CampoTipo.Nul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.Req)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.Req)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("TP_OPERACAO", CampoTipo.Req)]
        public int Tp_Operacao { get; set; }
        [Campo("TP_MODALIDADE", CampoTipo.Req)]
        public int Tp_Modalidade { get; set; }
        [Campo("TP_MODELONF", CampoTipo.Req)]
        public int Tp_Modelonf { get; set; }
        [Campo("CD_SERIE", CampoTipo.Req)]
        public string Cd_Serie { get; set; }
        [Campo("NR_NF", CampoTipo.Req)]
        public int Nr_Nf { get; set; }
        [Campo("TP_PROCESSAMENTO", CampoTipo.Req)]
        public string Tp_Processamento { get; set; }
        [Campo("DS_CHAVEACESSO", CampoTipo.Nul)]
        public string Ds_Chaveacesso { get; set; }
        [Campo("DT_RECEBIMENTO", CampoTipo.Nul)]
        public DateTime Dt_Recebimento { get; set; }
        [Campo("NR_RECIBO", CampoTipo.Nul)]
        public string Nr_Recibo { get; set; }
    }
}